addons/website_cf_turnstile/static/src/js/turnstile.js

```diff
@@ -13,14 +14,15 @@ publicWidget.registry.s_website_form.include({
             this.cleanTurnstile();
             if (!this.isEditable && !this.$('.s_turnstile').length && session.turnstile_site_key) {
                 const mode = new URLSearchParams(window.location.search).get('cf') == 'show' ? 'always' : 'interaction-only';
-                $(`<div class="s_turnstile cf-turnstile float-end"
-                         data-action="website_form"
-                         data-appearance="${mode}"
-                         data-response-field-name="turnstile_captcha"
-                         data-sitekey="${session.turnstile_site_key}"
-                         data-error-callback="throwTurnstileError"
-                    ></div>
-                    <script class="s_turnstile">
+                const turnstileContainer = renderToElement("website_cf_turnstile.turnstile_container", {
+                    action: "website_form",
+                    appearance: mode,
+                    additionalClasses: "float-end",
+                    errorGlobalCallback: "throwTurnstileErrorCode",
+                    sitekey: session.turnstile_site_key,
+                });
+                const turnstileScript = renderToElement("website_cf_turnstile.turnstile_remote_script");
+                const turnstileErrorGlobalScript = $(`<script class="s_turnstile">
```

Comment:
JQuery is still authorised on the frontend? This is more as to avoid adding some more when we could use something else.

Reply:
@thjo-odoo migrated the script to javascript only while still using a script element, just in case.

Feature seems to still work fine.